rs/protobuf/def/types/v1/consensus.proto

```diff
@@ -219,7 +219,18 @@ message GetIngressMessageInBlockResponse {
   bytes ingress_message = 1;
 }
 
-message StrippedBlockProposal {}
+message StrippedBlockProposal {
+  BlockProposal block_proposal_without_ingress_payload = 1;
+  repeated StrippedIngressMessage ingress_messages = 2;
+  ConsensusMessageId unstripped_consensus_message_id = 3;
+}
+
+message StrippedIngressMessage {
+  oneof msg {
+    bytes full = 1;
```

Comment:
Do we already need this? Since for now we only have ids.

Reply:
I actually strip ingress messages only if they have size above `1024` bytes (the value will probably change, after I do some benchmarks) so it is necessary.

I will do more benchmarks and maybe it will turn out that it's best if we either strip all ingress messages or none, in which case I will drop this option, but I prefer to have it now for the time being

Reply:
As discussed offline - I've ran several benchmarks and didn't observe any measurable improvements, so to make the code simple I removed the `oneof`.